syntax = "proto3";

package website_content.v1;

import "common.proto";

option go_package = "example.com/website_content/v1;website_content_v1";

message HeroItem {
  I18nString title = 1;
  I18nString subtitle = 2;
  CTA cta = 3;
}
